js/templates/spinner.svg

```diff
@@ -0,0 +1,16 @@
+<svg version="1.1" class="spinner<% if (ob.className) print(` ${ob.className}`) %>" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
While the spinner svg can be declared as an image url in the markup or CSS, that way won't allow you to colorize the spinner. So, it's preferable to use this template which is available as a template helper, for example:

```
<div><%= ob.spinner({ className: 'optionalClassname' }) %>
```

The css is such that the spinner will be inherited text color. So, if you put a clrT somewhere on the parent, the spinner will be that color.